pkg/repository/testworkflow/mongo.go

```diff
@@ -216,6 +216,30 @@ func (r *MongoRepository) GetRunning(ctx context.Context) (result []testkube.Tes
 	return
 }
 
+func (r *MongoRepository) GetFinished(ctx context.Context, filter Filter) (result []testkube.TestWorkflowExecution, err error) {
+	result = make([]testkube.TestWorkflowExecution, 0)
+	query, opts := composeQueryAndOpts(filter)
+	opts.SetSort(bson.D{{Key: "result.finishedAt", Value: -1}})
+	if r.allowDiskUse {
+		opts.SetAllowDiskUse(r.allowDiskUse)
+	}
+	query["$or"] = bson.A{
+		bson.M{"result.status": testkube.PASSED_TestWorkflowStatus},
+		bson.M{"result.status": testkube.FAILED_TestWorkflowStatus},
```

Comment:
so we don't count aborted, timed out, cancelled?

Reply:
That is a good question. Should a canceled or aborted test count as flaky?